src/components/ActionableInput/index.tsx

```diff
@@ -48,7 +49,8 @@ const ActionableInputContainer = styled.div`
 
 const ButtonContainer = styled(Button)<{ $wasClicked?: boolean }>`
   padding: 8px;
-  margin-left: 4px;
+  margin-left: -34px;
```

Comment:
negative margin is usually a smell. Using `position: relative` and setting `right` would be more natural.